app/uk/gov/hmrc/agentsubscriptionfrontend/controllers/SubscriptionController.scala

```diff
@@ -193,67 +201,11 @@ class SubscriptionController @Inject()(
   }
 
   val showLinkClients: Action[AnyContent] = Action.async { implicit request =>
-    appConfig.autoMapAgentEnrolments match {
-      case true =>
-        withSubscribingAgent { _ =>
-          withValidSession { (_, _) =>
-            toFuture(Ok(html.link_clients(linkClientsForm)))
-          }
-        }
-      case false => toFuture(InternalServerError)
-    }
+    toFuture(InternalServerError)
```

Comment:
is there any point of having this function any more? is this showLinkClients page part of auto mapping